lib/post_message_definition.yml

```diff
@@ -16,6 +16,14 @@ post_messages:
       ping:
         payload:
           <<: *user_session_properties
+      goBack:
+        warning: |
+          By passing your own method to this prop, you are overriding the
+          default navigation behaviour of the SDK which is to determine if widget was able
```

Comment:
\* default navigation behaviour of the SDK which is _used_ to determine if _the_ widget was able